internal/setup/setup_test.go

```diff
@@ -1688,6 +1686,59 @@ func (s *S) TestPackageMarshalYAML(c *C) {
 	}
 }
 
+func (s *S) TestPackageYAMLFormat(c *C) {
+	var input = map[string]string{
+		"chisel.yaml": string(defaultChiselYaml),
+		"slices/mypkg1.yaml": `
+			package: mypkg1
+			archive: ubuntu
+			slices:
+				myslice1:
+					contents:
+						/dir/file: {}
+		`,
+		"slices/mypkg3.yaml": `
+			package: mypkg3
+			archive: ubuntu
+			slices:
+				myslice:
+					essential:
+						- mypkg1_myslice1
```

Comment:
Can we please have another package entry with a global essentials list?   Does it remain global or is it unified during reading?  How is the output?

Are we missing any other cases?

Reply:
Global essentials are de-sugared into essentials for each slice so it does not remain global. We need a test for that.

Reply:
Discussed offline: agreed to keep it like this for now once we add the test mentioned above.

Reply:
Added new test. Thanks!